taccsite_cms/static/site_cms/css/src/_imports/components/bootstrap.form.css

```diff
@@ -15,6 +15,8 @@ Styleguide Components.Bootstrap.Form
 .form-control {
   border: var(--global-border--normal);
   border-radius: 0;
+
+  background-clip: border-box; /* to prevent background from showing through */
```

Comment:
**Why?** This was not a problem on default theme, but `has-dark-logo` allows this bug to be seen.